### PR TITLE
chore: remove unused deps

### PR DESCRIPTION
**Pull request by Uzlopak**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,12 +50,10 @@
     "@octokit/tsconfig": "^3.0.0",
     "@types/node": "^20.0.0",
     "@vitest/coverage-v8": "^2.0.2",
-    "axios": "^1.0.0",
     "esbuild": "^0.23.0",
     "express": "^4.17.1",
     "glob": "^11.0.0",
     "prettier": "3.3.3",
-    "prettier-plugin-packagejson": "^2.2.9",
```
Comment (wolfy1339) [inline package.json]: This is used by the CLI in the prettier workflow. you won't see it used in any scripts

Comment (Uzlopak) [reply package.json]: according to the documentation we never configured it accordingly

https://github.com/matzkoh/prettier-plugin-packagejson/blob/master/README.md#installation

Comment (Uzlopak) [reply package.json]: but if you are 100% sure that this plugin is in use i wll of course revert

Comment (wolfy1339) [reply package.json]: it was added in https://github.com/octokit/webhooks.js/pull/412

Comment (wolfy1339) [reply package.json]: You are correct that as it is, it is currently unused, due to Prettier v3.

The configuration should then be updated to add the plugin

Comment (Uzlopak) [reply package.json]: Done :)

**Review by wolfy1339 — APPROVED** [review]

**Pull request merged**